utils/browser.py

```diff
@@ -93,28 +97,37 @@ def start(webdriver_name=None, base_url=None, **kwargs):
         quit()
 
     browser_conf = conf.env.get('browser', {})
+    browser_kwargs = browser_conf.get('webdriver_options', {})
+    # Pull in browser kwargs from browser yaml
+    if 'sauce' in browser_conf:
+        webdriver_name = 'Remote'
+        browser_kwargs['command_executor'] = conf.cfme_data['sauce_ondemand']['url'].format(
+            conf.credentials['sauce']['sauce_user_name'],
+            conf.credentials['sauce']['sauce_api_key'])
 
     if webdriver_name is None:
         # If unset, look to the config for the webdriver type
         # defaults to Firefox
-        webdriver_name = browser_conf.get('webdriver', 'Firefox')
+        if browser_kwargs['desired_capabilities']['browserName'] == 'firefox':
+            webdriver_name = browser_conf['webdriver'] = 'Firefox'
+        else:
```

Comment:
Let's for completeness put an elif in here, let's say if it's ```chrome``` then set it to ```Chrome```, but then if it's not any of those things, we should log in the logger and "maybe" select Firefox as default.